notebooks/community/model_garden/model_garden_pytorch_timm.ipynb

```diff
@@ -0,0 +1,577 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #65.        search_algorithm=None,

You may want to not by setting to None, the default is Vizier
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1760/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Done - removed this parameter to use default algorithm.</p>